docker/pravega/Dockerfile

```diff
@@ -9,14 +9,16 @@
 #
 FROM openjdk:11.0.8-jre-slim
 
-RUN apt-get update && apt-get install -y -q \
+RUN mkdir -p /usr/share/man/man1 && apt-get update && apt-get install -y -q \
```

Comment:
Please bring `mkdir -p /usr/share/man/man1` to a line before rest of the CA certificates change so that the commands are collocated by functionality. 

Also, it is not intuitive why this change is needed. Could you please add a comment that explains it?

Reply:
Done.

Reply:
Agree with @ravisharda, it is unclear why we need `/usr/share/man/man1`.

Reply:
I have added comments explaining the inclusion of /usr/share/man/man1.